### PR TITLE
Add draft changelog for 2.6.1 release.

### PR DESCRIPTION
**Pull request by agorrod**

### CODE REVIEW

**Comment by agorrod** [comment]
@keithbostic @michaelcahill @sueloverso Would someone please review the draft changelog I put together for a 2.6.1 release.

I didn't think this release would have much in it, but there were over 1500 lines of commit log to sift through. We've been busy!

File: NEWS [hunk 1]
```diff
@@ -1,3 +1,100 @@
...
+
+* Add the ability to disable write ahead logging at a per-table granularity.
+  Accessed via log=(enabled) configuration for WT_SESSION::create API.
+  Partial logging has serious implications for recovery, it should be used
+  with caution.
+  refs #1989
+
+* Add the ability to asynchronously flush transactions via a new
+  WT_SESSION::transaction_sync API.
+  refs WT-1908, #1943
```
Comment (sueloverso) [inline NEWS]: This API doesn't flush the transactions, it waits for any previous asynchronous transaction flushes to complete.

File: NEWS [hunk 2]
```diff
@@ -1,3 +1,100 @@
...
+
+* Add the ability to create a named in-memory snapshot via a new
+  WT_SESSION::snapshot API.
+  refs WT-1839
+
+* Move the sync configuration setting from WT_SESSION::begin_transaction to
+  WT_SESSION::commit_transaction. Some applications don't know whether they
+  need to sync a transaction to disk until they come to commit.
```
Comment (sueloverso) [inline NEWS]: Not only did the `sync` configuration setting move, but it changed.  The `background` setting is new to sync this transaction in the background.  The `true` or `false` has been changed to `on` or `off`.

File: NEWS [hunk 3]
```diff
@@ -1,3 +1,100 @@
...
+* Improve how we decide whether to deepen a tree during an internal split
+  operation. Append workloads could create trees that were excessively deep.
+
+* Fix a bug where accessing an overflow item could return WT_NOTFOUND
+  incorrectly. The issue was related to an invalid transaction visibility
+  check.
+  refs WT-1745
+
+* Improve performance and avoid changing files on startup if no recovery is
+  required by by avoiding the creation of unnecessary checkpoints and log
```
Comment (sueloverso) [inline NEWS]: "by by"

File: src/docs/upgrading.dox [hunk 4]
```diff
@@ -1,5 +1,16 @@
 /*! @page upgrading Upgrading WiredTiger applications
 
+@section version_261 Upgrading to Version 2.6.1
+
+<dl>
+<dt>Move the per transaction sync configuration</dt>
+<dd>
+Move the sync configuration setting from WT_SESSION::begin_transaction to
+WT_SESSION::commit_transaction. Deprecated support for the old form is
+maintained for now.
```
Comment (sueloverso) [inline src/docs/upgrading.dox]: Just to repeat, it is both a move and a change from boolean to a string.

File: NEWS (hunk 2 above)
Comment (keithbostic) [inline NEWS]: Did it move? That is, it's not backward compatible?

File: NEWS [hunk 5]
```diff
@@ -1,3 +1,100 @@
...
+* Fix compiler warnings for LZ4 implementation on Windows.
+  refs #2006
+
+* Fix a performance issue with WT_SESSION::compact, where it would spend a
+  long time compacting tables that had no space to reclaim.
+  refs WT-1953
+
+* Fix a bug in the WiredTiger command line utility where it could create a
+  base configuration file for an existing database.
+  refs WT-1943
```
Comment (keithbostic) [inline NEWS]: Not sure if it's worth calling out, but this was really a family of bugs where we didn't get `WT_CONNECTION.is_new` "correct", that is, we didn't correctly figure out if we crashed in the middle of database creation.

File: NEWS [hunk 6]
```diff
@@ -1,3 +1,100 @@
...
+
+* Fix a build problem where recent versions of RedHat would fail to detect
+  posix_memalign presence correctly.
+  refs WT-1951
+
+* Fix several problems with how we create, recover and backup databases.
+  Related to order of creation and differences between Windows and Posix
+  file semantic differences.
```
Comment (keithbostic) [inline NEWS]: `Windows and Posix file semantic differences` -> `Windows and POSIX file system semantics`?

**Comment by keithbostic** [comment]
I called out some minor wordsmithing, but this looks fine to me.

File: NEWS (hunk 2 above)
Comment (sueloverso) [inline NEWS]: As was written in the upgrade page, deprecated support for the old form remains for now.

**Pull request merged**